packages/lib-react-components/src/AuthModal/components/RegisterForm/RegisterFormContainer.js

```diff
@@ -0,0 +1,94 @@
+import auth from 'panoptes-client/lib/auth'
+import { func, shape, string } from 'prop-types'
+import { useState } from 'react'
+
+import { useTranslation } from '../../../translations/i18n'
+import RegisterForm from './RegisterForm'
+
+const DEFAULT_HANDLER = () => true
+
+// The error handler in the panoptes-javascript-client still doesn't return the vanilla error message
```

Comment:
That comment was added in https://github.com/zooniverse/front-end-monorepo/pull/908,so there might be clues in that PR.

Reply:
I think it's referring to the case where a Panoptes response contains an array of errors, rather than a single error:
https://github.com/zooniverse/panoptes-javascript-client/blob/10e0ccf5c47bf7cb8e783c349075cbadccc6e22d/lib/api-client.js#L25-L35